### DISCUSSION TITLE
Help me please No VSti Instruments Work

### DISCUSSION THREAD

**Discussion by gatd** [apohl79/audiogridder#301] [Help]
Hi can someone please help me I can get vst2 and vst3 Effects working great but when I try to load as vst Instrument I immediately get "Error failed to add (whatever instrument I try to load) plugin! Error "
I have tried increasing the buffer size but it made no difference
thanks for your time

**Comment by apohl79** [comment]
What is the exact error? You can also check the server log.

**Comment by gatd** [comment]
Hi Andreas

Great software btw 
using parallels with windows 10 plus ag server 1.1 beta 10
I couldn’t  get the instruments to show at all at first , no gui "failed load yxxplugin !"all the time
managed to get it working by turning off core audio in logic quitting server relaunching core audio logic and restarting the server.
when the server crashes  which happens a lot  a server restart or quit and restart  won’t fix the problem and have to go through all the above steps again buffer is set to 20 btw
another problem is I am getting no midi through put when using vst3 soft tube instruments?

thanks for your help
kind regards



Simon Slater
gatdecor@gmail.com

**Reply by CaBuffalo** [reply]
"another problem is I am getting no midi through put when using vst3 soft tube instruments?" --- can you try VST format to see if it works?

**Comment by gatd** [comment]
yes vst works fine




Simon Slater
gatdecor@gmail.com

**Reply by CaBuffalo** [reply]
Yes, my curve bender had the same problem. I already reported it to Andreas.
